static/lessgap.js

```diff
@@ -76,6 +76,16 @@ class GaplessQueue {
     if (this.props.onPlayNextTrack) this.props.onPlayNextTrack(this.currentTrack);
   }
 
+  playTrack(idx) {
+    this.resetCurrentTrack();
+
+    this.state.currentTrackIdx = idx;
+
+    this.play();
+
+    if (this.props.onPlayNextTrack) this.props.onPlayNextTrack(this.currentTrack);
+  }
+
```

Comment:
Instead of this, would you mind just calling (from the app):

```
gaplessPlayer.gotoTrack(idx);

gaplessPlayer.play();
```

Alternatively you can add a second parameter to `gotoTrack` called `playImmediately` (boolean) which would call `this.play()` automatically at the end of the function. `gaplessPlayer.goToTrack(idx, true)`